src/member-api.js

```diff
@@ -54,42 +75,148 @@ export class MemberApi extends TypedEmitter {
    * @param {string} [opts.roleName]
    * @param {string} [opts.roleDescription]
    * @param {number} [opts.timeout]
-   *
-   * @returns {Promise<import('./generated/rpc.js').InviteResponse_Decision>}
+   * @returns {Promise<(
+   *   typeof InviteResponse_Decision.ACCEPT |
+   *   typeof InviteResponse_Decision.REJECT |
+   *   typeof InviteResponse_Decision.ALREADY
+   * )>}
    */
-  async invite(deviceId, { roleId, roleName, roleDescription, timeout }) {
+  async invite(
+    deviceId,
+    {
+      roleId,
+      roleName = ROLES[roleId]?.name,
+      roleDescription,
+      timeout = 1_800_000,
```

Comment:
This is 30 minutes. Is that a reasonable timeout?

Reply:
This seems too long maybe. My gut says 5 mins, but should check with Jen and others on the team who have done trainings, but we can adjust in the future. Can you move this to a `const DEFAULT_INVITE_TIMEOUT` at the top of the file?

Reply:
Lowered to 5 minutes and moved to a variable in 210ab8bca6ed1f415daf144396b64f8354a8d56d.

Reply:
I'll follow up on this number ~next week.